### PR TITLE
DAOS-4678 build: DAOS build with "COMPILER=covc" fails at GO build step

### PR DESCRIPTION
**Pull request by GitHuaKuang**
Excluded src/control from using Bullseye compiler covc.

Signed-off-by: Hua Kuang <hua.kuang@intel.com>

### CODE REVIEW

**Comment by sylviachanoiyee** [comment]
Will doing this exclude covc for Go build only? 
Is this component the last one to be built in the pipeline?
Can you check to make sure the rest of the build is run with covc?

File: src/control/SConscript [hunk 1]
```diff
@@ -122,6 +122,9 @@ def scons():
 
     denv = env.Clone()
 
+    if denv.get("COMPILER") == 'covc':
```
Comment (sylviachanoiyee) [inline src/control/SConscript:125]: Will doing this exclude covc for Go build only?
Is this component the last one to be built in the pipeline?
Can you check to make sure the rest of the build is run with covc?
There is a tool cov01 you can use to turn coverage on and off for certain parts of build.

Comment (jolivier23) [reply src/control/SConscript:125]: yes.  The denv is created on line 123 so this only affects stuff below this SConscript file

Comment (sylviachanoiyee) [reply src/control/SConscript:125]: Thanks

**Comment by GitHuaKuang** [comment]
The failures are known issues:
https://jira.hpdd.intel.com/browse/DAOS-4687
https://jira.hpdd.intel.com/browse/DAOS-3945

**Pull request merged**